functions/gcs/requirements.txt

```diff
@@ -0,0 +1 @@
+google-cloud-logging==1.6.0
```

Comment:
Where is this used?

Reply:
My bad. The sample was created when GCF Python cannot write stdout to Stackdriver Logging; I forgot to remove the reference after the support came online.